Source/Charts/Highlight/PieRadarHighlighter.swift

```diff
@@ -23,32 +23,30 @@ open class PieRadarHighlighter: ChartHighlighter
         let touchDistanceToCenter = chart.distanceToCenter(x: x, y: y)
         
         // check if a slice was touched
-        if touchDistanceToCenter > chart.radius
-        {
+        guard touchDistanceToCenter <= chart.radius else {
```

Comment:
guard or if here doesn't really make a difference.

Reply:
Again:
> You use a guard statement to require that a condition must be true in order for the code after the guard statement to be executed.

So in this case the guard makes it extra clear that the method can return early here